cli/objects/rule.py

```diff
@@ -369,16 +387,17 @@ def _get_ignore(self, rule_dict: dict[Any, Any]) -> bool:
         Returns:
             bool: A boolean value that determines if a firewatch rule is an ignore rule. True=ignore, False=don't ignore
         """
-        if "ignore" in rule_dict.keys():
-
-            ignore = rule_dict["ignore"]
-
-            if isinstance(ignore, str):
-                if ignore.lower() == "true":
-                    return True
-                else:
-                    return False
-            elif isinstance(ignore, bool):
-                return ignore
+        ignore = rule_dict.get("ignore", False)
 
-        return False
+        if isinstance(ignore, str):
+            if ignore.lower() == "true":
+                return True
+            else:
+                return False
+        elif isinstance(ignore, bool):
+            return ignore
+        else:
+            self.logger.error(
```

Comment:
if you must validate the use input I think it will be better to have one function that do the validation for the user input and not split it across multiple functions.

Reply:
I'd rather not do that in this PR. This can be a future change. I believe this is going to take quite a bit of work and doesn't really fit the purposes of this PR. I think we may be getting to far out of the scope of this specific PR.